docs/plans/GEMINI.md

```
@@ -0,0 +1,449 @@
+
+### 4.2 Bead Types
+
+| Type | Purpose |
+|------|---------|
+| `issue` | Single task or bug fix |
+| `convoy` | Multi-agent feature effort |
+| `methodology` | Process/methodology specification |
+
+### 4.3 Lifecycle Flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)